static/src/javascripts/projects/common/modules/navigation/newHeaderNavigation.js

```diff
@@ -6,8 +6,7 @@ import userAccount from 'common/modules/navigation/user-account';
 
 const enhanced = {};
 
-const getSidebarElement = (): ?HTMLElement =>
-    document.getElementById('main-menu');
+const getFlyIn = (): ?HTMLElement => document.getElementById('main-menu');
```

Comment:
Still not so sure about this name. Without you explaining it I can't visualise what "fly in" could mean. Event `flyInMenu` would help. Or what about `mainMenu` as that is what it is , no?